content/zh/blog/introducing_kthena.md

```diff
@@ -0,0 +1,140 @@
+
+## LLM 服务化的“最后一公里”困境
+
+大语言模型（LLM）正在以前所未有的速度重塑各行各业，但将其高效、经济地部署在生产环境中，特别是基于 Kubernetes 的云原生平台上，仍然困难重重。开发者们普遍面临以下挑战：  
+1. **资源利用率低**：LLM 推理，尤其是其独特的 KV Cache 机制，对 GPU、NPU 显存的占用是动态且巨大的。传统的负载均衡一般采用Round-Robin算法，无法感知这种负载特性，导致 GPU、NPU 资源闲置与请求排队并存，成本高昂。  
+2. **延迟与吞吐量难以兼顾**：LLM 推理分为“Prefill”（处理输入提示）和“Decode”（生成 Token）两个阶段，前者是计算密集型，后者是访存密集型。将两者混合调度，常常导致无法针对性优化，影响整体服务的响应速度和吞吐能力。因此PD分离的部署已经成为主流，但如何高效路由和调度，仍是一个难题。  
+3. **多租户与多模型管理复杂**：在企业环境中，通常需要同时提供多个不同模型、不同版本或经过 LoRA 微调的模型。如何实现请求的公平调度、优先级管理以及动态路由，是一个复杂的工程难题，业界甚至有些方案将AI网关与大模型一一对应。  
+4. **缺乏K8s原生集成**：许多现有的解决方案要么是外部系统，与 Kubernetes 生态割裂；要么过于复杂，无法满足生产级所需的简单易用性和灵活运维。
+
+## Kthena：云原生 LLM 推理的智能大脑
+
+为了攻克上述难题，Kthena 应运而生。它并非要取代现有的 LLM 服务框架（如 vLLM, sgLang），而是作为它们上层的智能“交通枢纽”和“调度中心”，深度集成于 Kubernetes 之中。
+
+<div style="text-align: center;"> {{<figure library="1" src="./kthena/kthena-arch.svg">}}
+</div>
+
+
+Kthena 的核心由两大组件构成：
+
+1）Kthena Router：一个独立、高性能面向多模型的router，负责接收所有推理请求，并根据 `ModelRoute` 规则，智能地将请求分发到后端的 `ModelServer`。
+
+2）Kthena Controller Manager：Kubernetes 控制平面的控制器，它主要包含多种控制器，负责 LLM 工作负载的编排与生命周期管理。它持续调谐并联动多类 CRD（如 `ModelBooster`、`ModelServing`、`AutoScalingPolicy`/`AutoScalingPolicyBinding`、以及 `ModelRoute`/`ModelServer`），将声明式API转化为运行时资源：ModelServing 控制器编排 `ServingGroup` 与 `Prefill/Decode` 角色分组；支持网络拓扑亲和调度和Gang调度、滚动升级与故障恢复；基于 `AutoScalingPolicy` 实现弹性扩缩容。
+
+这种架构使得 Kthena 成为连接用户请求与 LLM 模型的高度可编程的桥梁。
+
+## 核心特性与优势
+
+Kthena 的强大之处在于其专为 LLM 推理场景设计的核心功能：
+
+**1) 生产级推理编排（ModelServing）**
+
+<div style="text-align: center;"> {{<figure library="1" src="./kthena/model-serving.svg">}}
+</div>
+
+- **LLM工作负载三层架构设计**：ModelServing -> ServingGroup -> Role，一个API，支持LLM原生部署、PD分离部署，乃至大EP部署等多种部署形态，简化管理多LWS的负担。例如对于PD分离的大规模部署，可用一个ModelServing表示，根据负载的大小每个ModelServing可以包含任意数目的 ServingGroup（xPyD 分组）， 每个ServingGroup包含多个角色（Prefill Decode，他们通常部署在同一个超节点内以提升推理性能），相同的角色可以等价为一个LeaderWorkerSet，支持TP/PP/EP等多节推理并行计算。  
+- **原生支持Prefill-Decode分离部署**：将计算密集型的 Prefill 实例调度到配备高性能计算卡的节点组，而将访存密集型的 Decode 实例调度到配备高带宽显存的节点组，实现资源的最佳匹配和极致的端到端延迟优化。另可以独立伸缩，动态调整Prefill-Decode的比例，更灵活的应对各种复杂的业务场景（如长短句混合、实时推理等）。  
+- **多并行范式支持**：TP/PP/DP/EP 等并行模式灵活配置，最大化提升资源利用率和SLO  
+- **内置拓扑感知、Gang 调度支持**：Gang调度确保ServingGroup/Role“成组原子化”落地，避免资源浪费；拓扑感知调度通过将Role内的一组Pod调度到网络拓扑更优的节点，提升并行计算的数据传输时延。
+
+**2) 开箱即用的模型上线（ModelBooster）**
+
+- 针对主流的大模型，提供包括PD分离在内的多种部署范式模板，自动生成ModelRoute/ModelServer/ModelServing/Autoscaling等路由策略和生命周期管理资源  
+- 覆盖通用的部署场景，至于更灵活的编排可通过ModelServing进行细粒度的控制
+
+**3) 智能、模型感知的路由（Kthena Router）**
+
+- **多模型路由**：兼容OpenAI API，根据请求头或Body体内容，将流量调度到不同的基础模型。  
+- **插件化调度算法**：提供最少请求、最小时延、KV Cache 感知、Prefix Cache 感知、LoRA 亲和、GPU 利用率感知、公平调度等多种负载均衡算法，满足用户不同业务场景和部署形态的需求  
+- **LoRA 模型热插拔无中断**：感知推理引擎加载的LoRA 适配器，提供无中断的插拔和路由能力  
+- **丰富的流量治理策略**：基于权重的模型路由，金丝雀发布、Token级流控、故障转移
+- **All-in-one实现架构**，无需部署Envoy Gateway，原生支持PD分离的流量调度，将多层路由合并成一层，易于维护
+
+**4) 成本驱动的自动扩缩容（Autoscaler）**  
+- **同构伸缩**：支持稳定、突发双模式，按业务指标（CPU/GPU/内存/自定义）精准扩缩  
```

Comment:
Looks like a new line is needed here, as the current rendering is not showing the effect. Also same as point 5 and point 6:
<img width="993" height="342" alt="image" src="https://github.com/user-attachments/assets/e17e44d1-83a3-49c0-a0a2-8994ae775661" />

Reply:
fixed